xds/internal/resolver/xds_resolver_test.go

```diff
@@ -54,6 +54,15 @@ var (
 	target = resolver.Target{Endpoint: targetStr}
 )
 
+func TestRegister(t *testing.T) {
+	for _, s := range []string{"xds", "xds-experimental"} {
+		b := resolver.Get(s)
+		if b == nil {
```

Comment:
`if b == nil || (grpc.Version >= "1.30-dev" && s === "xds-experimental")`

OK that's probably not worth doing.

Reply:
What is it for?

Reply:
To ensure the "for a release" in:

> We will keep -experimental (so we have both) for a release.

Reply:
I just set a reminder to myself, to remove it week after the next release